sympy/plotting/tests/test_textplot.py

```diff
@@ -166,7 +172,9 @@ def test_imaginary():
         '      0 |_______________________________________________________',
         '         -1                         0                          1'
     ]
-    assert list(textplot_str(sqrt(x), -1, 1)) == lines
+    # RuntimeWarning: invalid value encountered in sqrt
+    with ignore_warnings(RuntimeWarning):
+        assert list(textplot_str(sqrt(x), -1, 1)) == lines
```

Comment:
Maybe the plotting module should use `np.errstate` if it expects to deliberately pass invalid values:
https://numpy.org/doc/stable/reference/generated/numpy.errstate.html